netket/operator/_hamiltonian.py

```diff
@@ -379,6 +369,11 @@ def __init__(
                        at every odd site of the lattice. For non-bipartite lattices, the
                        sign rule cannot be applied. Defaults to True if the lattice is
                        bipartite, False otherwise.
+         acting_on_subspace: Specifies the mapping between nodes of the graph and
+            Hilbert space sites, so that graph node :code:`i ∈ [0, ..., graph.n_nodes - 1]`,
+            corresponds to :code:`acting_on_subspace[i] ∈ [0, ..., hilbert.n_sites]`.
+            Must be a list of length `graph.n_nodes`. Passing a single integer :code:`start`
+            is equivalent to :code:`[start, ..., start + graph.n_nodes - 1]`.
```

Comment:
```suggestion
             acting_on_subspace: Specifies the mapping between nodes of the graph and
                Hilbert space sites, so that graph node :code:`i ∈ [0, ..., graph.n_nodes - 1]`,
                corresponds to :code:`acting_on_subspace[i] ∈ [0, ..., hilbert.n_sites]`.
                Must be a list of length `graph.n_nodes`. Passing a single integer :code:`start`
                is equivalent to :code:`[start, ..., start + graph.n_nodes - 1]`.
```